### ISSUE TITLE
Migrations naming consistency is unclear

### ISSUE THREAD

**Issue by caherrerapa** [malparty/keywords#41]
I had problems identifying which is the file naming convention on the migrations. Also, while checking the file content the class names also don't follow a consistent pattern all over. e.g. sometimes it's camelcase, sometimes all lowercase. I'm not a .Net expert, but afaik all classes should be Pascal case.
<img width="434" alt="keywords_KeywordsApp_Migrations_at_main_·_malparty_keywords" src="https://user-images.githubusercontent.com/1084920/118337907-0a947280-b53f-11eb-9d67-836c5cfb5333.png">

**Comment by malparty** [comment]
Hi,
Thanks for the feedback. Indeed I worked for 7 years on a Db-first project, so I did not experimented much with code first db migrations. 
Here in this project I learned that the migration file name is reused for naming the classes generated by `dotnet-ef` tool. 

Indeed, [C# code convention](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions) recommend PascalCase for classes, struct, public properties, etc...

I will then refactor the code to follow Nimble Compass conventions ([from Ruby on Rails doc](https://nimblehq.co/compass/development/code-conventions/ruby-on-rails/#relational-database)).

Pull-request coming soon~ 
EDIT: Pull Request here #48
